src/components/molecules/PaginationButtons/PaginationButtons.types.ts

```diff
@@ -1,4 +1,5 @@
 export type PaginationButtonProps = {
+  forcePage: number
```

Comment:
Why is it called `forcePage`? For me it's not descriptive and names should be

Reply:
forcePage is built in attribute in ReactPaginate component, for me it also is not descriptive, but it's the only way I guess